### PR TITLE
Header: Restore default iFixit logo

### PR DESCRIPTION
**Pull request by ianrohde**
## Issue

We're restoring the standard iFixit logo on the header.

## CR/QA

Straight-forward change, but let's make sure there are no regressions.

`Wordmark` component contains the standard iFixit logo SVG.

Linked to https://github.com/iFixit/ifixit/issues/51066

### CODE REVIEW

**Comment by ianrohde** [comment]
deploy_block 🔴 on syncing with https://github.com/iFixit/ifixit/pull/51454

File: frontend/app/(defaultLayout)/components/page-frame/header.tsx [hunk 1]
```diff
@@ -82,7 +81,7 @@ export function Header({ menu }: React.PropsWithChildren<HeaderProps>) {
                   title="iFixit turns 20"
```
Comment (hcutrone) [inline frontend/app/(defaultLayout)/components/page-frame/header.tsx:81]: I'm pretty sure the title should be removed here, too. And maybe the padding
```suggestion
```

**Review by hcutrone — COMMENTED** [review]
CR 🌯 but dev_block 🌯 on removing an extra `title` attribute

File: frontend/app/(defaultLayout)/components/page-frame/header.tsx (hunk 1 above)
Comment (ianrohde) [reply frontend/app/(defaultLayout)/components/page-frame/header.tsx:81]: Both should go, thanks. 👍

Comment (ianrohde) [reply frontend/app/(defaultLayout)/components/page-frame/header.tsx]: https://github.com/iFixit/react-commerce/pull/2195/commits/137aeba2a7166e198ae49964043944e03396d78b

**Review by hcutrone — APPROVED** [review]
CR 🌯

**Comment by deltuh-vee** [comment]
QA 🎬 
iFixit Logo has been restored in the header.

**Comment by deltuh-vee** [comment]
un_dev_block 👍

**Comment by ianrohde** [comment]
un_deploy_block 🟢

**Pull request merged**